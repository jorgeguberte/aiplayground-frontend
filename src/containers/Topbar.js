import React from 'react'
import ReactDOM from 'react-dom'

import '../styles/Topbar.css'

export default class Topbar extends React.Component{
    render(){
        return(
            <div className="Topbar row">
                <span className="Topbar-title">AIPLAYGROUND</span>
            </div>
        )
    }
}

