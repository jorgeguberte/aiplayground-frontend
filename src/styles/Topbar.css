.Topbar{
    background-color: #363f45;
    color: white;
    padding: 1em;
}
/*.Topbar span{
    font-size: 1.1em;
    text-shadow: 1px 1px 1px rgba(0,0,0,0.004);
    text-rendering: optimizeLegibility;
    letter-spacing: 0.1em;
}*/