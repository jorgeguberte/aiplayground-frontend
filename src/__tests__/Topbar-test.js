import React from 'react'
import {shallow, mount, renbder} from 'enzyme'
import renderer from 'react-test-renderer'

import Topbar from '../containers/Topbar';


describe('integrity', ()=>{   
    test('should render', ()=>{
        const component = renderer.create(<Topbar/>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    const wrapper = mount(<Topbar/>);

    test('it has a title', ()=>{
        const title = wrapper.find('.Topbar-title');
        expect(title.text()).toBe('AIPLAYGROUND');
    })
});
