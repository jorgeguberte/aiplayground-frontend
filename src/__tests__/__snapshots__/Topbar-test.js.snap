// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`integrity should render 1`] = `
<div
  className="Topbar row"
>
  <span
    className="Topbar-title"
  >
    AIPLAYGROUND
  </span>
</div>
`;
